BangazonAPI/Controllers/CustomerController.cs

```diff
@@ -0,0 +1,214 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Threading.Tasks;
+using BangazonAPI.Models;
+namespace BangazonAPI.Controllers
+
+{
+    [Route("api/Customer")]
+
+    [ApiController]
+
+    public class CustomerController : ControllerBase
+    {
+        private readonly IConfiguration _config;
+
+        public CustomerController(IConfiguration config)
+        {
+            _config = config;
+        }
+        public SqlConnection Connection => new SqlConnection(_config.GetConnectionString("DefaultConnection"));
+        public async Task<IActionResult> Get()
+        {
+            using (SqlConnection conn = Connection)
+            {
+                List<object> customerControlerlist = new List<object>();
```

Comment:
Why is this a `List<object>`?

Reply:
the generic form of any given object for use with multiple objects

Reply:
fixed

Reply:
But you know the true data type of each list element. Please use the correct type.